react/index.tsx

```diff
@@ -123,13 +130,18 @@ const RichText: FunctionComponent<Props> = ({
     'textPosition'
   )
 
-  const html = insane(marked(text), sanitizerConfig)
+  const html = insane(
+    //TODO: While markdown component isn't released, it needs to be done this way.
```

Comment:
Any issue created related to this?

Reply:
Not that sure, but @Jeymisson told me to put this `TODO` there

Reply:
There should be an issue. Those TODOs will not be taken into account unless someone sees them by chance in the future.